### PR TITLE
2968 - Change how we load in Google Analytics

### PR DESCRIPTION
**Pull request by aliuk2012**
We used to automatically add in Google Analytics (GA), we now 
have to ask users for consent before adding cookies to their devices.
At the same time we should only inject GA if the user consents
otherwise they are downloading more javascript that will not be
used or needed.

This commit removes GTM (we decided to not replace GA with GTM).
GA should still load as before but we will add the cookie policy
control to this to ensure it only gets used if the user consents
to cookies

### CODE REVIEW

File: app/webpacker/scripts/analytics.spec.js [hunk 1]
```diff
@@ -7,6 +10,45 @@ describe("Analytics", () => {
...
+      expect(document.querySelectorAll('script[src]')).toHaveLength(1)
+    })
+
+    it ('takes uses a default tracking ID', () => {
+      loadAnalytics()
+      const createGAInstanceWithID = ga.mock.calls[0][1]
+      expect(createGAInstanceWithID).toEqual('UA-112932657-1')
+    })
+
+    it ('takes accepts a custom tracking ID', () => {
```
Comment (fofr) [inline app/webpacker/scripts/analytics.spec.js]: `takes` or `accepts`

**Comment by fofr** [comment]
Does this PR have the cookie check, or will that come later?

File: app/webpacker/scripts/analytics.spec.js (hunk 1 above)
Comment (aliuk2012) [reply app/webpacker/scripts/analytics.spec.js]: Good spot 😄 👍

**Comment by aliuk2012** [comment]
@fofr  Still to come. I've broken up the PR so that its easier to review instead of a one large PR.


I'll create a label for cookies and label each PR relating to the cookies work

**Comment by aliuk2012** [comment]
[[M] Build and release new cookie banner and cookie policy page.](https://trello.com/c/OnWGvVre/2968-m-build-and-release-new-cookie-banner-and-cookie-policy-page)

**Pull request merged**